test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -92,6 +91,51 @@ static public void testCreatePoint2D() throws Throwable {
 		assertEquals(getY.invoke(point2D), yNew);
 	}
 
+	/*
+	 * Create a safe value type with multibyte primative members
+	 * 
+	 * value Point2DSafe {
```

Comment:
what is the meaning of safe? Did you mean Point2DComplex?

Reply:
Updated to state it is verified, rather than the arbitrary use of safe (not defined using `Unsafe`)